### DISCUSSION TITLE
List in file as input in scraper pro

### DISCUSSION THREAD

**Discussion by van-geaux** [omkarcloud/google-maps-scraper#160] [Q&A]
Is it possible to implement a list of links/queries in file like txt to upload to the webui as input to start the scraping process?

For example, if I want to scrape restaurants in multiple cities (city A to city J) by subdistrict (subdistrict Aa-Bz per city), instead of putting the queries one by one, it would be awesome to put the queries I want in a txt file to upload to webui instead putting each queries in the list one by one to the webui

I have quite a bit of queries to do. I have read the discussion in https://github.com/omkarcloud/google-maps-scraper/discussions/81 but I understand almost none of it. I understand a bit of python and I understand how to do loops, I've taken a look at the code but I don't know where to put the list because I don't understand where the entire process start

**Comment by Theophilesncfscrap** [comment]
up

**Comment by Chetan11-dev** [comment]
Done!

**Reply by Osushi** [reply]
It was the most ideal! Thank you!

**Reply by luuminhvuong93** [reply]
how . is there a tutorial or guideline on how to do this

**Reply by Chetan11-dev** [reply]
Click Bulk Add and paste the file contents
